### PR TITLE
DOCS-2212: Update outdated control panel screenshots

### PR DESCRIPTION
**Pull request by sguequierre**
* still needs input controller check (uses webgamepad screenshot and I don't have access to one in texas: created [this ticket](https://viam.atlassian.net/browse/DOCS-2212#:~:text=contributed%20to%20by-,DOCS%2D3066,-Check%20if%20webgamepad) to track)

### CODE REVIEW

File: assets/components/camera/example_camera_image.png
Comment (sguequierre) [inline assets/components/camera/example_camera_image.png:1]: I can recreate this with the little guy when back in NYC

File: docs/fleet/control.md [hunk 1]
```diff
@@ -23,9 +23,7 @@ For local communication between [parts](/architecture/parts/#machine-parts) Viam
 
 {{<gif webm_src="/fleet/control.webm" mp4_src="/fleet/control.mp4" alt="Using the control tab" max-width="800px">}}
 
-You can also switch between different machine parts and control the selected machine part directly from the **CONTROL** tab.
-
-{{< imgproc src="/fleet/app-usage/control-machine-part-dropdown.png" alt="The expanded Machine part dropdown menu with the main part selected on the Control tab." resize="650x" class="aligncenter" >}}
+You can also switch between different machine parts directly from the **CONTROL** tab and control the selected machine part.
```
Comment (sguequierre) [inline docs/fleet/control.md:26]: figured this should be self-explanatory by app but still worth a sentence to explain feature

Comment (sguequierre) [reply docs/fleet/control.md:26]: same above

File: static/fleet/control.gif
Comment (sguequierre) [inline static/fleet/control.gif:1]: for some context rover camera was not displaying properly, the sunlight was too bright so everything was white

Comment (npentrel) [reply static/fleet/control.gif:1]: good but please compress the gif a bit more

**Review by npentrel — APPROVED** [review]
LGTM % GIF compression. Thanks!

**Review by npentrel — APPROVED** [review]

**Pull request merged**